crates/typst-library/src/text/raw.rs

```diff
@@ -92,6 +93,8 @@ pub struct RawElem {
     /// )
     ///
     /// With `rg`, you can search through your files quickly.
+    /// This example searches the current directory recursively
+    /// for the text ``` Hello World```:
```

Comment:
```suggestion
    /// for the text ```Hello World```:
```

Reply:
this would format `World` with the language `Hello`. I almost did it that way, but then checked what it did.

I could point that out in the description, what do you think?

Reply:
Oh, I didn't know about that possibility. I think you could point that out if you want